src/commands/webViewCommand.ts

```diff
@@ -114,7 +114,7 @@ export class WebViewCommand {
         let masked = undefined;
         try {
           masked = await cx.mask(result.filename);
-          this.logs.info("Masked Secrets by AI Guided Remediation: " + (masked && masked.maskedSecrets ? masked.maskedSecrets.length : "0"));
+          this.logs.info("Masked Secrets by AI Security Champion: " + (masked && masked.maskedSecrets ? masked.maskedSecrets.length : "0"));
```

Comment:
Let's add the "AI Security Champion" to a constant in messages.ts and use it in .ts files. So that it makes it easier to handle/change, since it noticed it is all over the project.

Reply:
I created in under utils/common/constants.ts that contains the contants for the whole project.
Maybe at a later stage we can break it down to app domains if we see fit..